src/content/docs/kubernetes-pixie/k8s-otel/install.mdx

```diff
@@ -0,0 +1,635 @@
+
+### Configure using NRDOT [#configure-nrdot]
+
+1. Download the [Helm chart values file](https://github.com/newrelic/helm-charts/tree/master/charts/nr-k8s-otel-collector/values.yaml#L20-L24) adapt it to meet your specific requirements. For more information on the parameters, refer to the [parameters in `value.yaml` file](#config-params).
+
+   * Cluster name and <InlinePopover type="licenseKey"/> are mandatory.
+
+   * Check the entire list of [configuration parameters](https://github.com/newrelic/helm-charts/tree/master/charts/nr-k8s-otel-collector#values).
+
+2. Install the [Helm chart](https://github.com/newrelic/helm-charts/tree/master/charts/nr-k8s-otel-collector) together with the values file.
+
+    ```shell
+    helm repo add newrelic https://helm-charts.newrelic.com
+    helm upgrade nr-k8s-otel-collector newrelic/nr-k8s-otel-collector -f your-custom-values.yaml -n newrelic --create-namespace --install
+    ```
+
+3. Ensure the pods have been successfully spun up.
+
+    ```shell
+        kubectl get pods -n newrelic --watch
+    ```
+
+    You should see pods with names like `nr-k8s-otel-collector-<hash>` in the `newrelic` namespace.
+
+4. Make sure New Relic is getting the data it needs, including metrics, events, and logs, by running the right queries. For more information, refer to [Introduction to the query builder](/docs/query-your-data/explore-query-data/query-builder/introduction-query-builder/).
+
+    ```sql
+    FROM Metric SELECT * WHERE k8s.cluster.name='<CLUSTER_NAME>'
+    FROM InfrastructureEvent SELECT * WHERE k8s.cluster.name='<CLUSTER_NAME>'
+    FROM Log SELECT * WHERE k8s.cluster.name='<CLUSTER_NAME>'
+    ```
+
+
+### Configure using your own OpenTelemetry Collector [#configure-own-otel]
+
+If you prefer to use your own OpenTelemetry Collector, you can set it up to collect Kubernetes telemetry data. Ensure that your collector includes the necessary components for Kubernetes monitoring, as outlined in the [Requirements](#requirements) section.
+
+To configure your own OpenTelemetry Collector to send data to New Relic:
+
+  1. If you're deploying your custom collector via our Helm chart, update the `image` settings in your `values.yaml` file to specify your desired OpenTelemetry Collector Contrib image:
+
+    ```yaml
+    image:
+        repository: otel/opentelemetry-collector-contrib # Example: Using the contrib distro
+        tag: "latest" # Or a specific stable version like "0.98.0"
+    ```
+
+    If you are deploying your collector manually, ensure you have the `opentelemetry-collector-contrib` binary or container image available.
```

Comment:
Binary image should be available in their container registry.
I don't think we need this sentence